src/screens/MainScreen/index.js

```diff
@@ -31,11 +60,26 @@ const MainScreen = ({ getTargets, targets, loading, error }) => {
 
   const markers = useMemo(() => renderMarkers(), [targets]);
 
+  const renderCurrentLocation = () => {
+    if (location.latitude && location.longitude) {
```

Comment:
```suggestion
    const { latitude, longitude } = location;
    if (latitude && longitude) {
```

Reply:
Is it possible to destructure `location` when getting it from `useState`?
```suggestion
const [{ latitude, longitude }, setLocation] = useState({});
```

I'm not asking to change this, I'm just curious

Reply:
I tried this, and yes, it works!!! I'll keep things the old way since I find the code more readable that way.